perf/sawtooth/sawtooth_workload/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "sawtooth_workload"
+version = "0.1.0"
+authors = ["Peter Schwarz <pschwarz@bitwise.io>"]
```

Comment:
Is authors a required field? We have generally scrubbed this info/used generic contact emails for this in other langs.